docs/examples/embed.js

```diff
@@ -0,0 +1,62 @@
+/*
+	An example of how you can send embeds
+*/
+
+// Extract the required classes from the discord.js module
+const { Client, MessageEmbed } = require('discord.js');
+
+// Create an instance of a Discord client
+const client = new Client();
+
+// The ready event is vital, it means that your bot will only start reacting to information
+// from Discord _after_ ready is emitted
+client.on('ready', () => {
+  console.log('I am ready!');
+});
+
+client.on('message', message => {
+  // If the message is "how to embed"
+  if (message.content === 'how to embed') {
+    // We can create embeds using the MessageEmbed constructor
+    // Read more about all that you can do with the constructor in the docs
+    const embed = new MessageEmbed()
+      .setTitle(`A slick little embed`) // Set the title of the field
+      .setColor('RANDOM') // Set the color of the embed
+      .setDescription(`Hello, this is a slick embed!`); // Set the main content of the embed
```

Comment:
the comments in this entire section are like a wall of text that makes it seem more complex, consider more spacing, and get rid of the inline comments
and change the set color to take a hex code, `RANDOM` isn't explanatory unless you've already read the section of colors in the docs